src/Extension/WorkflowedElement.php

```diff
@@ -29,6 +31,30 @@ public function updateCMSFields(FieldList $fields)
         }
     }
 
+    /**
+     * Whether the current user is an admin and is performing an
+     * inline (graphql) publish on an individual element.
+     *
+     * @param Member $member The current user
+     * @return boolean
+     */
+    public function canInlinePublish($member = null)
+    {
+        if (!Permission::checkMember($member, 'ADMIN')) {
+            return false;
+        }
+
+        try {
+            $req = Controller::curr()->getRequest();
+            $data = json_decode($req->getBody());
+            $pub_action = ($data->operationName == 'PublishBlock');
+            $cur_block = ($data->variables->blockId == $this->owner->ID);
```

Comment:
Preferably these'd be camelCase rather than snake_case as they're not statics (no need to change here)

Reply:
Roger that!